src/LazyOperations/MinkowskiSum.jl

```diff
@@ -211,6 +211,30 @@ function constraints_list(ms::MinkowskiSum)
     return constraints_list(minkowski_sum(ms.X, ms.Y))
 end
 
+"""
+    ∈(x::AbstractVector{N}, ms::MinkowskiSum{N, <:Singleton, <:LazySets}) where {N}
+
+Check whether a given point is contained in the minkowski sum of a singleton
+and a lazy set.
+
+### Input
+
+- `x`  -- point (any vector type)
```

Comment:
```suggestion
- `x`  -- point
```
(just to be consistent with other methods)

Reply:
Ok, if you want to call it "inconsistency" this text in parentheses; i think that it helps, because we don't have point type.